.editorconfig

```
@@ -0,0 +1,23 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = false
+max_line_length = 140
```

Our stylecheck sets this to 100 (though only as a warning, not an error).
```suggestion
max_line_length = 100
```